src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -46,16 +46,16 @@
           @on-selected-change="toggleSelectRequirement"
         />
       </div>
+      <a
+        v-if="!editMode"
+        class="newCourse-link"
+        @click="toggleEditMode()"
+        @keyup.enter="toggleEditMode()"
+        tabindex="0"
+      >
+        Edit Requirements
+      </a>
     </div>
-    <a
-      v-if="!editMode"
-      class="newCourse-link"
-      @click="toggleEditMode()"
-      @keyup.enter="toggleEditMode()"
-      tabindex="0"
-    >
-      Edit Requirements
-    </a>
```

Comment:
can you explain why moving this magically fixes the problem?

Reply:
it looks like the previous div only renders if `"nonAutoRequirements.length > 0"` (line 16)

Reply:
holddd uppp let me look into it a little more